src/traits/from-iterator.md

```diff
@@ -0,0 +1,23 @@
+# FromIterator
```

Comment:
You should also mention this page in `SUMMARY.md` to actually make it appear somewhere :smile: